addons/pinniped/post-deploy/pkg/configure/configure_test.go

```diff
@@ -88,6 +87,20 @@ func TestPinniped(t *testing.T) {
 		},
 	}
 
+	pinnipedInfoConfigMapWorkloadCluster := &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: "kube-public",
+			Name:      "pinniped-info",
+		},
+		Data: map[string]string{
+			"cluster_name":                         "",
```

Comment:
on the workload cluster, i think it would be neater if these empty fields didn't even show up in the `ConfigMap`. i know this means the code is more complex, but i think it makes the `pinniped-info` API cleaner which might help us avoid weird edge cases (e.g., does an empty `"issuer"` mean that we aren't setting that field on the workload cluster, or is the supervisor issuer actually empty for some reason). wdyt?

i.e., this
```yaml
apiVersion: v1
kind: ConfigMap
data:
  pinniped_api_group_suffix: pinniped.dev
  pinniped_concierge_is_cluster_scoped: "false"
...
```
instead of this
```yaml
apiVersion: v1
kind: ConfigMap
data:
  cluster_name: ""
  issuer: ""
  issuer_ca_bundle_data: ""
  pinniped_api_group_suffix: pinniped.dev
  pinniped_concierge_is_cluster_scoped: "false"
...
```

Reply:
Updated it, lmk what you think!